python/ray/data/dataset.py

```diff
@@ -942,7 +945,8 @@ def repartition(self, num_blocks: int, *, shuffle: bool = False) -> "Dataset":
                 minimizing data movement.
 
         Returns:
-            The repartitioned dataset.
+            The repartitioned :class:`Dataset`. After repartitioning, all blocks have
+            approximately the same number of rows.
```

Comment:
is the all blocks have approximately the same number of rows true? I don't think it is right?

Reply:
Not sure. That's what our old documentation said.